Dockerfile

```diff
@@ -47,9 +48,14 @@ ARG GO_STRIP
 ARG CGO_ENABLED
 # VERSION sets the version for the produced binary
 ARG VERSION
-RUN --mount=ro --mount=type=cache,target=/root/.cache \
+# GOVERSIONINFO_VERSION defines goversioninfo tool version
+ARG GOVERSIONINFO_VERSION
+RUN --mount=type=bind,target=.,rw \
```

Comment:
Do you know if there's a way to store the generated files "elsewhere"? (e.g., somewhere in a `temp` directory)? Thinking: if we can keep them elsewhere, we wouldn't have to deal with "rw" mounts, as well as `.gitignore/.dockerignore` etc.

Reply:
Yes I would like to use a tmpfs but the syso has to be in the source dir. Will check again if there is a way to change that in rsrc.

Reply:
Ok was able to change paths so we keep the tmpfs.